packages.html

```diff
@@ -0,0 +1,334 @@
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-53797708-5', 'auto');
+  ga('send', 'pageview');
+
+</script>
+<script type="application/javascript" src="//www2.usgs.gov/scripts/analytics/usgs-analytics.js"></script>
+</head>
+
+<body>
+
+	<div id="header"></div>
+
+	<div id="nav"></div>
+
+<!-- BEGIN Main Content -->
+
+<main>
+
+    <article>
+    
+        
+      <h1 class="page-title-h1">USGS-R Packages</h1>
+        
+  		<section class="box content" id="package_intro">
+
+<p>Clear communication about package expectations is very important. Package developers should be transparent about the maintenance, development, and user support associated with their package so that potential users are aware. To help with this communication for USGS R packages, we have created the following categories:</p>
+
+<ul>
+  <li><a href="#core">Core</a></li>
+  <li><a href="#research">Research</a></li>
+  <li><a href="#support">Support</a></li>
+  <li><a href="#orphan">Orphan</a></li>
+</ul>
+<p>
+Each are described in detail below. In general, they go from most support and outreach ("Core") to least ("Orphan"). See also the section in our "R-package development" training: <a href="https://owi.usgs.gov/R/training-curriculum/r-package-dev/maintenance/">maintenance</a> for a deeper conversation on package considerations.
+
+</p>  
+              
+      </section>
+		
+      <section class="box content" id="core">
+          
+              <h4>Core</h4>
+<p>
+A package with the designation "Core" has several key features:</p>
+
+<div id="plain">
+<ul>
+  <li>Critically important to a wide and general audience</li>
+  <li>Published, peer-reviewed documentation on the package function</li>
+  <li>Long-term support addressed publicly (could be stated in README or elsewhere)</li>
+  <li>Rapid response to bug fixes</li>
+  <li>Rapid response to R version upgrades</li>
+  <li>Rapid response to R package dependency changes</li>
+  <li>Customer support and troubleshooting</li>
+  <li>An active USGS employee is designated as maintainer. Development and maintenance of core packages should include a team of developers. Maintainers coordinate and lead the development team, but should not be the sole developer.</li>
+  <li>The package should be well tested, and the package should be tested on a continuous integration platform regularly.</li>
+  <li>All external functions should have clear documentation and working examples.</li>
+</ul>
+</div>
+
+<p>
+A user of core packages can expect prompt feedback from bug reports and general-use questions. Users can expect smooth transitions of maintainers since the package is well-tested, well-documented, and multiple developers have collaborated on the project. Enhancements should be carefully considered, "scope-creep" should be actively avoided (that is, careful thought to add new features). The general scope of the package should not change over time.  </p>            
```

Comment:
`careful thought to add new features` -> `give careful thought to adding new features`